cmd/root.go

```diff
@@ -11,7 +11,15 @@ import (
 var rootCmd = &cobra.Command{
 	Use:   "clconf [global options] command [command options] [args...]",
 	Short: `A utility for merging multiple config files and extracting values using a path string`,
-	RunE:  getv,
+	Long: `A utility for merging multiple config files and extracting values using a path string
+			the order of precedence from least to greatest is
+				--yaml
+				YAML_FILES
+				--yaml-base64
+				YAML_VARS
+				Standard Input
```

Comment:
probably want to say `--stdin` to show that you have to specify the option (like you do with the others).  Also, is the arbitrary indentation here some sort of trial and error to match some level of indent?  does the lib not handle indent (one of the CLI libs i use in one of the langages i use handles that automatically, but may be c#)

Reply:
[Fixed](https://github.com/pastdev/clconf/pull/16/commits/93a001f6441aa3013080a3d152dff92ae24058e8)